templates/jecoute/surveys_list.html.twig

```diff
@@ -55,14 +58,17 @@
                 {%- spaceless %}
                     {% for data in surveys %}
                         {% set survey = data[0] %}
+                        {% set zone = survey is instanceof('App\\Entity\\Jecoute\\LocalSurvey') ? survey.zone %}
 
                         {% set row = {
                             id: survey.id,
                             name: survey.name,
                             createdAt: survey.createdAt|date('d/m/Y'),
                             questionsCount: data.questions_count,
                             responsesCount: data.responses_count,
-                            zone: survey is instanceof('App\\Entity\\Jecoute\\LocalSurvey') ? survey.zone|trans : null,
+                            zone: survey is instanceof('App\\Entity\\Jecoute\\LocalSurvey')
+                                ? '<span class="adherent-dashboard-tag">'~('geo_zone.'~zone.type)|trans~'</span><br>'
+                                    ~"<div class=\"b__nudge--top-10\">#{zone.name} (#{zone.code})</span>",
```

Comment:
what will happen if the zone variable is null? is this possible?

Reply:
For `LocalSurvey`, no.

Reply:
so this template displays only LocalSurvey? I ask for this code: `#{zone.name} (#{zone.code})` which does not work if zone is null